framework/interface/templates/target_manager.html

```diff
@@ -121,23 +130,37 @@
                     targets_search_api_url:"{{ targets_search_api_url }}",
                     targets_ui_url:"{{ targets_ui_url }}",
                     plugins_api_url:"{{ plugins_api_url }}?group=web&group=network",
-                    worklist_api_url:"{{ worklist_api_url }}"
+                    worklist_api_url:"{{ worklist_api_url }}",
+                    checked_rows:[] //Track checked rows for bulk operations
```

Comment:
This is used to keep the constants. Move the variable `checked_rows` to somewhere else.